docs/topics/db/models.txt

```diff
@@ -154,7 +154,7 @@ ones:
     <Field.blank>`, the field will be required.
 
 :attr:`~Field.choices`
-    An :term:`iterable` of 2-tuples to use as choices for this field. If this
+    A :term:`sequence` of 2-tuples to use as choices for this field. If this
```

Comment:
Looks good, but I think that is just not enough for an issue described in the ticket, i.e.
```python
ACTIVITY_TYPES = set(['Create', 'Update', 'Delete'])
...
field = models.CharField(
     choices=[(x,x) for x in ACTIVITY_TYPES],
     ...
```
in this case `choices` is a list but an order is changing. I would add a note:
```
.. note::
    A new migration is created each time an order of ``choices`` changes.
```